lib/elixir/src/elixir_expand.erl

```diff
@@ -8,7 +8,13 @@
 expand({'=', Meta, [Left, Right]}, E) ->
   assert_no_guard_scope(Meta, "=", E),
   {ERight, ER} = expand(Right, E),
-  {ELeft, EL}  = elixir_clauses:match(fun expand/2, Left, E),
+  {ELeft, EL} = elixir_clauses:match(fun expand/2, Left, E),
+  if
+    element(1, ELeft) == '<<>>' ->
+      refute_parallel_bitstring_match(ERight, E, ?key(E, context) == match);
```

Comment:
I think you only need to call `refute_parallel_bitstring_match` only if the context is match. That's because if the context is not a match, you would have already called it when expanding the inner pattern. Your tests would be able to say if this is true or not. :)

Reply:
Yeah, I tested this already, it does not work.